datasets/selfdet.py

```diff
@@ -59,6 +62,12 @@ def __init__(self, root, detection_transform, query_transform, cache_dir=None, m
                     self.files.append(path)
                 else:
                     continue
+        if self.pseudo_labels in ['mdetr', 'mdef_detr']:
+            # Load mdetr or mdef_detr boxes
+            pseudo_dets_dets_file_path = f"{root}_{self.pseudo_labels}_dets.pkl"
```

Comment:
Let's have a different `.pkl` file name for each model

Reply:
The pickle files will be named, `train_mdetr_dets.pkl` and `train_mdef_detr_det.pkl`.